include/hip/hcc_detail/code_object_bundle.hpp

```diff
@@ -124,6 +126,18 @@ class Bundled_code_header {
 
                 std::copy_n(f + y.header.offset, y.header.bundle_sz, std::back_inserter(y.blob));
 
+                // we need to read the blob (ELF) for this bundle to get the feature flags
+		std::stringstream tmp{std::string{y.blob.cbegin(), y.blob.cend()}};
+                ELFIO::elfio reader;
+                if (reader.load(tmp)) {
+			unsigned int xnack_enabled = reader.get_flags() & 0x100;
+			unsigned int sram_ecc_enabled = reader.get_flags() & 0x200;
+			if (xnack_enabled)
+				y.triple.append("+xnack");
+			if (sram_ecc_enabled)
+				y.triple.append("+sram-ecc");
+		}
```

Comment:
indentation of this new block of code is off

Reply:
done.

Reply:
nope, line 130 is still misaligned and the number of space characters for the indent is different to other places in this same function